### DISCUSSION TITLE
Öğrenci kaydı

### DISCUSSION THREAD

**Discussion by Erkam80** [community/community#30646] [GitHub Education]
Merhabalar, github öğrenci kaydı için belge yüklüyorum ancak size iletilmiyor. Bilgilerimi işleme koyamıyorum. Yardımcı olur musunuz

**Comment by thomas-ayissi** [comment]
Bonjour,
Genel olarak, belgeleri içeren bir posta gönderdiğinizde, aşağıdaki metni içeren bir posta alırsınız:

``````
GitHub Eğitim avantajları için başvurunuzu gönderdiğiniz için teşekkür ederiz. Başvurunuzu aldık ve TARİH itibariyle bizden bir e-posta alacaksınız.

Başvurunuzun mevcut durumunu kontrol etmek için lütfen https://education.github.com/benefits adresini ziyaret edin. Başvurunuz hala incelenmeyi bekliyorsa, sayfanın sağındaki kenar çubuğunun en üstünde görünecektir. Başvurunuz zaten onaylanmışsa, GitHub Eğitim avantajlarınıza yönlendirileceksiniz.

Herhangi bir sorunuz varsa, https://github.community/t/github-student-developer-pack-faq/124041 adresindeki SSS bölümümüzü ziyaret edin.

Octotastic bir gün geçirin!

- GitHub Eğitim Ekibi
``````

Özetleyelim:
1/ Başvurunuzu https://education.github.com/ web sitesinden yaptığınızı kontrol edin

Not: Prosedürün tamamı şu adreste ayrıntılı olarak açıklanmıştır: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student

2/ Belgelerinizin alındığını bildiren e-postayı arayın. "edu-noreply@github.com" adresinden gelmelidir.

3/ "edu-noreply@github.com" adresinden gelen yanıt e-postasını okuyun. Ya size başvurunuzun "Github Education" için kabul edildiğini söyleyecektir. Ya da posta size başvurunuzun neden reddedildiğini tam olarak açıklayacaktır. Yapmanız gereken tek şey, talep edilen belirli belgeleri sağlamaktır ve her şey yolunda gitmelidir.

**Comment by thomas-ayissi** [comment]
Yukarıda verilen Eğitim Github sitesine giderseniz ve belgeleriniz yüklenmezse.

Kontrol etmeyi dene:

1/ Belgeleri talep edilen formatta yüklediğinizi. Örneğin, sizden PDF isteniyorsa ve siz JPG yüklemek istiyorsanız, bu işe yaramayabilir.

2/ Dosyalarınız Github Eğitim sayfasında belirtilen maksimum boyutu aşıyor mu? Dosyalarınız çok büyükse gönderilmeyecektir.

3/ Mümkünse, dosyalarınızı akşam saatlerinde göndermeye çalışın, çünkü internet bağlantısı genellikle daha yüksek hıza sahiptir. 

4/ Belgeleri akıllı telefondan değil bilgisayardan göndermeyi de unutmayın. Daha büyük ekran, dosyalarınızı göndermeye çalıştığınızda görünebilecek hata mesajını daha iyi okumanızı sağlayacaktır. Hata mesajında istenilenleri yapmanız sorunlarınızı çözecektir.

İyi akşamlar dilerim!

**Comment by samus-aran** [comment]
Hi @Erkam80, thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
